firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionException.java

```diff
@@ -83,11 +83,11 @@ public enum Status {
     /**
      * The Firebase App Distribution Tester API is disabled for this project.
      *
-     * <p>The developer of this app must enable the API in the Google Cloud Console before using the
-     * App Distribution SDK. See the <a
-     * href="https://firebase.google.com/docs/app-distribution/set-up-alerts?platform=android">documentation</a>
-     * for more information. If you enabled this API recently, wait a few minutes for the action to
-     * propagate to our systems and retry.
+     * <p>Before you use the App Distribution SDK in your app, you must enable the API in the Google
```

Comment:
Could we directly link to the right place in pantheon here? I've seen that in other error messages.